tests/test_modeling_tf_blenderbot_small.py

```diff
@@ -0,0 +1,265 @@
+# coding=utf-8
+# Copyright 2021 The Facebook, Inc and The HuggingFace Inc. team. All rights reserved.
```

Comment:
As Sylvain likes to ask, did the Facebook Inc authors really contribute to this test file?